python/jupyter_leaflet/src/Map.ts

```diff
@@ -234,6 +235,14 @@ export class LeafletMapView extends LeafletDOMWidgetView {
     const view = await this.create_child_view<LeafletControlView>(child_model, {
       map_view: this,
     });
+    if (
+      view instanceof LeafletGeomanDrawControlView &&
```

Comment:
I don't like much making a special case here. We may want to find a better approach.

Though let's not consider this a blocker for merging the PR. Can you at least add a comment e.g. "TODO: Remove this special case"

Reply:
I totally agree with you. Unfortunately geoman constructs and adds it's toolbar in it's own way, which means if we want to not have the special case here, it looks like we would have to reconstruct the whole toolbar ourselves.

Reply:
Thanks for adding a comment 👌🏽  

Other question, if you remove the control Python side, does it work as intended? We may need a special case in the removal logic too?

Reply:
Removal should work correctly now, although (at least as far as I can see) we don't need a special case, since we can just do this in the `remove` method of the class. Let me know if you foresee any issues with doing it this way!

Reply:
Only geoman should add stuff there, since `L.PM` is added by geoman itself (apparently before becoming `leaflet-geoman`, the package was known as `leaflet.pm`)